xla/backends/gpu/codegen/copy_test.cc

```diff
@@ -0,0 +1,169 @@
+/* Copyright 2025 The OpenXLA Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#include "xla/backends/gpu/codegen/copy.h"
+
+#include <gtest/gtest.h>
+#include "xla/hlo/ir/hlo_instructions.h"
+#include "xla/tests/hlo_test_base.h"
+
+namespace xla {
+namespace gpu {
+namespace {
+
+using CopyFusionTest = HloTestBase;
```

Comment:
HloTestBase is deprecated (see message attached to the class). I think you can use HloHardwareIndependentTestBase instead.

Reply:
Done.